spec/unit/mutant/expression/method_spec.rb

```diff
@@ -30,19 +29,16 @@
       let(:input) { instance_method }
 
       it 'returns correct matcher' do
-        should eql(
-          Mutant::Matcher::Method::Instance.new(
-            env,
-            TestApp::Literal, TestApp::Literal.instance_method(:string)
-          )
-        )
+        expect(subject.to_a.map(&:expression)).to eql([object])
       end
     end
 
     context 'with a singleton method' do
       let(:input) { singleton_method }
 
-      it { should eql(Mutant::Matcher::Method::Singleton.new(env, TestApp::Literal, TestApp::Literal.method(:string))) }
+      it 'returns correct matcher' do
+        expect(subject.to_a.map(&:expression)).to eql([object])
```

Comment:
Is it necessary to call `#to_a` and then `#map` here? If it is enumerable then `#map` should be supported natively.